### PR TITLE
feat(core): update migration state after db init

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

After database init (`createTables()`), update migration state to latest migration file. Then the database is ready for the next migration.

notice: the word "migration" will be changed later.

## Testing

截屏2022-09-26 下午12 02 23

截屏2022-09-26 下午12 00 48

### CODE REVIEW

File: packages/core/src/database/seed.ts [hunk 1]
```diff
@@ -79,6 +80,9 @@ export const createDatabaseCli = async (dsn: string) => {
       await pool.query(sql`${raw(query)}`);
       console.log(`${chalk.blue('[create-tables]')} Run ${file} succeeded.`);
     }
+
+    await updateDatabaseTimestampToLatest(pool);
+    console.log(`${chalk.blue('[create-tables]')} Update migration state succeeded.`);
```
Comment (gao-sun) [inline packages/core/src/database/seed.ts:85]: alteration?

File: packages/core/src/database/seed.ts [hunk 2]
```diff
@@ -79,6 +80,9 @@ export const createDatabaseCli = async (dsn: string) => {
       await pool.query(sql`${raw(query)}`);
       console.log(`${chalk.blue('[create-tables]')} Run ${file} succeeded.`);
     }
+
+    await updateDatabaseTimestampToLatest(pool);
```
Comment (gao-sun) [inline packages/core/src/database/seed.ts:84]: or just `updateDatabaseAlterationTimestamp`? we can add a timestamp param later, and if no param is specified then use the latest

File: packages/core/src/migration/index.ts [hunk 3]
```diff
@@ -70,6 +70,18 @@ export const updateDatabaseTimestamp = async (pool: DatabasePool, timestamp: num
   );
 };
 
+export const updateDatabaseTimestampToLatest = async (pool: DatabasePool) => {
+  const files = await getMigrationFiles();
+
+  const latestFile = files[files.length - 1];
+
+  if (files.length === 0 || !latestFile) {
```
Comment (gao-sun) [inline packages/core/src/migration/index.ts:78]:
```suggestion
  if (!latestFile) {
```

should be fine?

**Review by gao-sun — APPROVED** [review]

File: packages/core/src/database/seed.ts (hunk 1 above)
Comment (wangsijie) [reply packages/core/src/database/seed.ts:85]: "migration" will be renamed in LOG-4244

File: packages/core/src/database/seed.ts (hunk 2 above)
Comment (wangsijie) [reply packages/core/src/database/seed.ts]: updated

File: packages/core/src/database/seed.ts (hunk 1 above)
Comment (gao-sun) [reply packages/core/src/database/seed.ts:85]: cool

**Pull request merged**